subsys/bluetooth/services/fast_pair/fp_crypto/fp_crypto_common.c

```diff
@@ -130,23 +130,30 @@ size_t fp_crypto_account_key_filter_size(size_t n)
 }
 
 int fp_crypto_account_key_filter(uint8_t *out, const struct fp_account_key *account_key_list,
-				 size_t n, uint8_t salt)
+				 size_t n, uint8_t salt, const uint8_t *battery_info)
```

Comment:
Maybe we could call it `battery_notification` (since the data is actually whole battery_notification from spec)?

Reply:
Same with macro naming

Reply:
Discussed offline. Let's leave it as it is.